dhis-2/dhis-services/dhis-service-core/src/test/java/org/hisp/dhis/program/notification/ProgramNotificationServiceTest.java

```diff
@@ -720,7 +720,8 @@ private void setUpInstances() {
                 userLvlTwoLeftRight,
                 userRoot));
     userGroupBasedOnParent =
-        createUserGroup('H', Sets.newHashSet(userLvlTwoLeftLeft, userLvlTwoLeftRight));
+        createUserGroup(
+            'H', Sets.newHashSet(userLvlTwoLeftLeft, userLvlTwoLeftRight, userLvlOneLeft));
```

Comment:
I'm not familiar with this code, but does adding the third user in this set cover the case where the base org unit will receive a notification?

Reply:
@muilpp `testSendToParent` would fail after the change I have made. In order to test the changed functionality `userLvlOneLeft` user was added in UserGroup `userGroupBasedOnParent`